$OpenBSD: patch-cvsgraph_c,v 1.1 2015/08/21 07:03:05 jasper Exp $

cvsgraph.c:193: warning: format '%d' expects type 'int', but argument 3 has type 'size_t'
cvsgraph.c:194: warning: format '%d' expects type 'int', but argument 3 has type 'size_t'

--- cvsgraph.c.orig	Fri Aug 21 08:49:55 2015
+++ cvsgraph.c	Fri Aug 21 08:51:48 2015
@@ -190,8 +190,8 @@ static void dump_dtext(char *p, dtext_t *d)
 {
 	printf("%sdtext.rev  : ", p);
 	dump_rev("", d->rev);
-	printf("%sdtext.log  : %d bytes\n", p, d->log ? strlen(d->log) : -1);
-	printf("%sdtext.text : %d bytes\n", p, d->text ? strlen(d->text) : -1);
+	printf("%sdtext.log  : %ld bytes\n", p, d->log ? strlen(d->log) : -1);
+	printf("%sdtext.text : %ld bytes\n", p, d->text ? strlen(d->text) : -1);
 	printf("\n");
 }
 
